src/checkEnvFile.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+const checkEnvFile = (): void => {
+  const envFilePath = path.join(process.cwd(), '.env');
+
+  try {
+    // Check if the .env file exists
+    fs.accessSync(envFilePath, fs.constants.R_OK);
+    console.log('.env file found!');
+  } catch (err) {
+    console.error(
+      'Error: .env file not found. See 1Password under "ssportal-env" or see https://github.com/IABTechLab/uid2-self-serve-portal?tab=readme-ov-file#environment-variables'
```

Comment:
I know I personally said to mention 1password in this check, but thinking about it now I don't love putting this in something public.  Maybe just reference the readme?

Reply:
yeah agreed - done!